lib/registries/addon/overview/controller.ts

```diff
@@ -85,4 +85,3 @@ export default class Overview extends Controller {
         + (this.registration.relatedCounts.linkedRegistrations || 0);
     }
 }
-
```

Comment:
Same here. If you're not working on this file, we should minimize changes for removing new lines.

Reply:
That sounds good. Thank you Yuhuai.

Reply:
Yuhuai, removing these lines at the end of the file results in the GItHub actions lint portion of the tests failing. How would you recommend proceeding?

Reply:
I have added a space and lint on tests are now passing.